django_x509/templates/admin/django_x509/change_form.html

```diff
@@ -22,4 +22,56 @@
     </div>
 </fieldset>
 {% endif %}
+
+<script>
```

Comment:
create a JS file for this and load it like is done here:
https://github.com/openwisp/django-loci/blob/master/django_loci/base/admin.py#L42-L44